.github/workflows/lint.yml

```diff
@@ -0,0 +1,11 @@
+on: pull_request
+name: Lint check
+jobs:
+  black:
+    name: Lint check
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Black Code Formatter"
+        uses: "lgeiger/black-action@v1.0.1"
+        with:
+          args: ". --check"
```

Comment:
I am not sure this is getting the proper path 🤔 

![image](https://user-images.githubusercontent.com/2728804/81560136-2d59dd00-9367-11ea-9539-b64d3a859024.png)